src/app/organization/[orgSlug]/admin/settings/organization-settings-manager.tsx

```diff
@@ -0,0 +1,33 @@
+'use client'
+
+import { Paper } from '@mantine/core'
+import { useDisclosure } from '@mantine/hooks'
+import { OrganizationSettingsEdit } from './organization-settings-edit'
+import { OrganizationSettingsDisplay } from './organization-settings-display'
+import { type Org } from '@/schema/org'
+
+interface OrganizationSettingsManagerProps {
+    org: Org
+}
+
+export function OrganizationSettingsManager({ org }: OrganizationSettingsManagerProps) {
+    const [isEditing, { open: startEdit, close: cancelEdit }] = useDisclosure(false)
+
+    const handleSaveSuccess = () => {
+        cancelEdit()
+    }
+
+    return (
+        <Paper shadow="xs" p="xl" mb="xl">
+            {isEditing ? (
+                <OrganizationSettingsEdit
+                    org={org}
+                    onSaveSuccess={handleSaveSuccess}
```

Comment:
since `handleSaveSuccess` only calls `cancelEdit` this could be:
```suggestion
                    onSaveSuccess={cancelEdit}
```

Reply:
yes, I'm also seeing that right now in my editor. Makes no sense to use that method